data/archiver/archiver.py

```diff
@@ -0,0 +1,102 @@
+import os
+import logging
+from data.archiver.aws_s3_client import AwsS3
+from data.archiver.ftp_uploader import FtpUploader
+from data.archiver.utils import compress, md5
+from data.archiver.dataclass import DataArchiverRequest, DataArchiverResult
+from data.archiver.ingest_api import Ingest
+
+STREAM_ARCHIVE = False
```

Comment:
Is this part of the config?

Reply:
Not global var anymore. Stream option now obtained in request.